### PR TITLE
Added Generic Assests

### PR DESCRIPTION
**Pull request by miketamis**
# View Asset Balance
```gql
{
  network(network: CENNZnet_Nikau) {
    address(address: "5G6L4FqzKtpe9T48eUK5fTrfM9znsmPGKNrxcynaw2hCfRoL") {
     	balance(assetId: 17008) {
        amount
        asset {
          symbol
        }
      }
     
    } 
  }
}
```

# Create Asset
```gql
{
  network(network: CENNZnet_Nikau) {
    address(address: "5G6L4FqzKtpe9T48eUK5fTrfM9znsmPGKNrxcynaw2hCfRoL") {
     	createNewGenericAsset(options: {
        initialIssuance: 10000,
        allowBurn: "Self",
        allowMint: "Self",
        allowUpdate: "Self"
      }, metadata:{
        symbol: "BEAR",
        decimalPlaces: 0,
      }) {
        id
        signerPayload
      }
     
    } 
  }
}
```

# Create Transfer Transcation

```gql
{
  network(network: CENNZnet_Nikau) {
    address(address: "5G6L4FqzKtpe9T48eUK5fTrfM9znsmPGKNrxcynaw2hCfRoL") {
     	balance(assetId: 17008) {
        createTransferTransaction(toAddress: "5CniLEDp3csvYuYCkKjsPiZ9cf4e5MVEckkf1SB4sB4QG8YJ", amount: 10) {
          id
          signerPayload
        }
      }
     
    } 
  }
}
```

### CODE REVIEW

File: entities/BalanceEntity.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        outputType: "Balance",
+      });
+    },
+  },
+});
+
+/*
+{
+  "data": {
+    "submitTransaction": {
```
Comment (p-dealwis) [inline entities/BalanceEntity.ts]: delete?

File: entities/GenericAssetEntity.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      targetAddress: string;
+      amount: number;
+    }) {
+      const api = await this.apiConnector();
+
+      const permission = await permissions(this);
+      const burnAddress = permission.V1.burn.Address;
+      return this.createTransaction({
+        address: burnAddress,
+        extrinsic: api.tx.genericAsset.mint(
```
Comment (p-dealwis) [inline entities/GenericAssetEntity.ts:83]: should be burn?

Comment (miketamis) [reply entities/GenericAssetEntity.ts]: fixed

**Review by p-dealwis — APPROVED** [review]
LGTM!

**Pull request merged**